app/client/sass/desktop_view.scss

```diff
@@ -3,6 +3,11 @@
    flex-direction: row;
   }
 
+  .headshots {
+    height: 300px;
```

Comment:
You'd better be positive that these images are square, otherwise they will be distorted. It would be safer to simply specify the `width`, not both dimensions.